apps/et/et-cos/demo.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   releaseName: et-cos
   values:
     java:
-      image: hmctspublic.azurecr.io/et/cos:pr-1787-ab4ae6a-20240417094144 #{"$imagepolicy": "flux-system:demo-et-cos"}
+      image: hmctspublic.azurecr.io/et/cos:latest #{"$imagepolicy": "flux-system:et-cos"}
```

Comment:
Dont change the bit after the : the flux bot will do that for you (I imagine that's where your conflict came from). Just change the demo-et-cos to et-cos.